src/bin/frontendlauncher

```diff
@@ -138,6 +142,7 @@ do
         lang=$arg
         export LANG="$lang"
         export LANGUAGE="$lang"
+	export LC_NUMERIC="C"
```

Comment:
This sneaked in, is this a difference between 3.3RC and develop?

Reply:
This may have been added to deal with commas versus dots in numeric values for the international community. I think we may have been outputting commas in the xml files and when we read them back in it caused parsing errors. This may have been put in to solve this.

Reply:
This was contributed by cessenat and he put it on develop and we didn't add it to the RC as well.

Reply:
thanks for the context Eric!